plugins/zulip/alerta_zulip.py

```diff
@@ -33,17 +27,36 @@
 ZULIP_TO = app.config.get('ZULIP_TO') \
            or os.environ.get('ZULIP_TO')
 ZULIP_SUBJECT = app.config.get('ZULIP_SUBJECT') \
-             or os.environ.get('ZULIP_SUBJECT', 'Alert')
+                or os.environ.get('ZULIP_SUBJECT', 'APImon alert')
```

Comment:
Maybe create common function for getting such variables instead of using multiline strings?